tests/functions.py

```diff
@@ -435,6 +435,18 @@ def numpy_arctan(a):
   return np.arctan(a)
 
 
+def numpy_atleast_1d(a):
+  return np.atleast_1d(a)
+
+
+def numpy_atleast_2d(a):
+  return np.atleast_2d(a)
+
+
+def numpy_atleast_3d(a):
+  return np.atleast_3d(a)
```

Comment:
If the input variable to a test function is `x`, we pass in an array. I think this is what you want to do. Then, the output should probably be a scalar, so return `np.sum(np.atleast_*d(x))`

Reply:
Although if the tests pass without wrapping it with `np.sum`, that is fine, too.